data/.gitignore

```diff
@@ -0,0 +1 @@
+/raw
```

Comment:
Why is only the raw folder in this gitignore? If this is in, must we then also put the others in there?

Reply:
shouldn't we do it the same way as the interim and processed folders?

Reply:
Fixed it by moving it to the raw dir and iognoring the datafiles, rather than the directory itself